docs/pages/installation.mdx

```diff
@@ -15,66 +15,7 @@ up-to-date information.
 
 (!docs/pages/includes/permission-warning.mdx!)
 
-<Tabs>
-  <TabItem label="Debian/Ubuntu (DEB)">
-    ```bash
-    # Install our public key.
-    curl https://deb.releases.teleport.dev/teleport-pubkey.asc | sudo apt-key add -
-    # Add repo to APT
-    add-apt-repository 'deb https://deb.releases.teleport.dev/ stable main'
-    # Update APT Cache
-    apt-get update
-    # Install Teleport
-    apt install teleport
```

Comment:
This step is different than the one we use on the Getting Started page. It will actually prevent me from installing Teleport on Ubuntu 20+